addon/validators/format.js

```diff
@@ -76,7 +79,12 @@ export default Base.extend({
     const { regex, type } = options;
 
     if (type && !isNone(regularExpressions[type]) && isNone(regex)) {
-      options.regex = regularExpressions[type];
+      if (type === 'email' && options.allowNonTld) {
+        options.regex = regularExpressions['emailNonTld'];
```

Comment:
As `emailNonTld` is a literal string, please replace it with `options.regex = regularExpressions.emailNonTld`